src/utils/IVeriCredential.ts

```diff
@@ -1,6 +1,9 @@
 export interface CredentialSubject {
-	name: object[];
-	birthDate: object[];
+	name?: object[];
+	birthDate?: object[];
+	device_information?: {
```

Comment:
This is interesting - the change should only be for TxMA events and this type is also used for VCs.

I think we probably want to make a `TxMACredentialSubject` that uses  `CredentialSubject` but also had the `device_information`?

Reply:
Like the update - it can probably live with the other TxMA types in `src/utils/TxmaEvent.ts` though?